package.json

```diff
@@ -22,25 +22,24 @@
     "middleware"
   ],
   "dependencies": {
+    "@fastify/busboy": "^1.0.0",
     "append-field": "^1.0.0",
-    "busboy": "~0.3.1",
     "concat-stream": "^2.0.0",
     "fastify-plugin": "^2.0.1",
-    "mkdirp": "^0.5.1",
+    "mkdirp": "^1.0.4",
```

Comment:
This feels like too small of a change to make a separate PR, but with Node version bump to 10 it is possible to use newer mkdirp.